packages/components/src/card/card-divider/hook.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../../ui/context';
+import * as styles from '../styles';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../../divider').DividerProps, 'hr'>} props
+ */
+export function useCardDivider( props ) {
+	const { className, ...otherProps } = useContextSystem(
+		props,
+		'CardDivider'
+	);
+
+	const classes = useMemo(
+		() =>
+			cx(
+				styles.Divider,
+				styles.borderColor,
+				'components-card__divider',
+				className
+			),
+		[ className ]
+	);
+
+	return {
+		...otherProps,
+		className: classes,
+		children: null,
```

Comment:
Is passing `children` as `null` really necessary?

Reply:
I was trying to make sure that `Divider` wouldn't render any `children` at all by forcing that value to be `null`, but I guess it's not really necessary (plus, we have types in place to warn against it).

Removed in https://github.com/WordPress/gutenberg/pull/32566/commits/b67dd3f7ff7148cb7308867bb6e3da9e3d662802